tests/Saver/MongoTest.php

```diff
@@ -4,16 +4,16 @@ class Saver_MongoTest extends PHPunit_Framework_TestCase
 {
     public function testSave()
     {
-        $data = file_get_contents(XHGUI_ROOT_DIR . '/tests/fixtures/results.json');
+        $data = json_decode(file_get_contents(XHGUI_ROOT_DIR . '/tests/fixtures/results.json'), true);
 
-        $profiles = $this->getMockBuilder('Xhgui_Profiles')
+        $collection = $this->getMockBuilder(MongoCollection::class)
             ->disableOriginalConstructor()
             ->getMock();
-        $profiles->expects($this->once())
+        $collection->expects($this->once())
             ->method('insert')
-            ->with($this->equalTo($data));
+            ->with($this->equalTo($data + ['_id' => Xhgui_Saver_Mongo::getLastProfilingId()]));
 
-        $saver = new Xhgui_Saver_Mongo($profiles);
+        $saver = new Xhgui_Saver_Mongo($collection);
```

Comment:
Shouldn't this test move to the collector long term? Right now this is a test for a class from another package.

Reply:
Definitely! However I don't have enough privileges to set up Travis for that project, so I didn't bother. But it should be done.